### ISSUE TITLE
iOS - Click on marker causing crash

### ISSUE THREAD

**Issue by aperdec** [Fulanko/reactxp-map#3]
The code used below is working on android and on web, however on iOS devices, when the user selects one of the markers, the application displays an error which I have attached below. 

```
<ReactXPMap
    style={cs.styles.propertyMapPanelMap}
    zoom={13}
    mapType="roadmap"
    latitude={51.0486}
    longitude={-114.0708}
    showLocation={true}
    geolocation={true}
    locationText="You are here"
    apiKey="**********">
    {this.props.assets.map(asset => <Marker
        key={asset.Id}
        latitude={asset.Lat}
        longitude={asset.Long}
        title={asset.AssetName}
        color="1DA9F9"
        onPress={this._showLocationInformation}/>)}
</ReactXPMap>
```

![image](https://user-images.githubusercontent.com/13382495/41300610-61da1be6-6e23-11e8-854a-707b9022793b.png)

**Comment by Fulanko** [comment]
Thanks for reporting this. This is due to a bug in react-native-maps. (https://github.com/react-community/react-native-maps/issues/1499)

I fixed it in version 0.1.7 (56fc0cefa082507bd3fb977c2f873c9258002d3d).

**Comment by aperdec** [comment]
Tested with the new version and it worked. Thanks for the quick fix.
